stable/yugaware/templates/rbac.yaml

```diff
@@ -7,6 +7,23 @@ metadata:
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 {{- if .Values.rbac.create }}
+{{- if .Values.ocpCompatibility.enabled }}
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-cluster-monitoring-view
```

Comment:
probably not a good idea to include this because it will fail for users with the default permissions right? We could add a subflag under ocpCompatibility like 

```
ocpCompabiliity:
   enabled: true
   metrics_enabled: true
```

Reply:
The initial thought was, in those cases they can just set 
```
rbac:
  create: false
```